Add support for SQL Server connection URI (#859)
eclipse-vertx/vertx-sql-client: vertx-mssql-client/src/main/asciidoc/index.adoc

{@link examples.MSSQLClientExamples#reconnectAttempts}
----

> === connection URI

Reviewer: ```suggestion
=== Connection URI
```

> Apart from configuring with a `MSSQLConnectOptions` data object, We also provide you an alternative way to connect when you want to configure with a connection URI:

Reviewer: ```suggestion
As an alternative to configuring the client with a {@link io.vertx.mssqlclient.MSSQLConnectOptions} data object, you can use a connection URI:
```

[source,$lang]
----
{@link examples.MSSQLClientExamples#configureFromUri(io.vertx.core.Vertx)}
----

> The connection URI format is defined by the client in an idiomatic way: `sqlserver://[user[:[password]]@]host[:port][/database][?attribute1=value1&attribute2=value2...`

Reviewer: ```suggestion
The connection URI format is defined by the client in an idiomatic way: `sqlserver://[user[:[password]]@]host[:port][/database][?attribute1=value1&attribute2=value2...]`
```
